src/app/components/organism/timeline-experiency/timeline-experiency.component.css

```
@@ -1,119 +1,79 @@
+  z-index: -1;
+}
+
+/* Spine fill — progressive, driven by scroll (scaleY set inline) */
+.timeline-progress {
+  position: absolute;
+  width: 2px;
+  height: 100%;
+  top: 0;
+  left: 50%;
+  margin-left: -1px;
+  transform-origin: top;
```

Stylelint (17.14.0)

[error] Duplicate property "transform-origin" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)